environment/src/main/java/com/sequenceiq/environment/environment/validation/network/azure/AzureExistingPrivateDnsZonesService.java

```diff
@@ -6,27 +6,53 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Service;
 
+import com.sequenceiq.cloudbreak.cloud.azure.AzurePrivateDnsZoneDescriptor;
+import com.sequenceiq.cloudbreak.cloud.azure.AzurePrivateDnsZoneRegistrationEnum;
 import com.sequenceiq.cloudbreak.cloud.azure.AzurePrivateDnsZoneServiceEnum;
 import com.sequenceiq.environment.network.dto.NetworkDto;
 
 @Service
 public class AzureExistingPrivateDnsZonesService {
 
-    public Map<AzurePrivateDnsZoneServiceEnum, String> getExistingZones(NetworkDto networkDto) {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AzureExistingPrivateDnsZonesService.class);
+
+    public Map<AzurePrivateDnsZoneServiceEnum, String> getExistingServiceZones(NetworkDto networkDto) {
```

Comment:
the method naming is
- not clear for me, would prefer something like `getExistingManagedPrivateDnsZones`
- seems to be generic tough the implementation is only for RDS (logging implies generic implementation)

I would stick with this `managed` phase instead of  calling it `service`, generally

Reply:
So according to you can the two types be named as:
- ExistingManagedPrivateDnsZones: those zones that is registered and used by cloudbreak
- ExisstingRegisteredPrivateDnsZones: those zones that are only registered by cloudbreak, but is not used within CB?

Reply:
Sounds perfect!